libindy/tests/metrics.rs

```diff
@@ -25,13 +20,31 @@ mod collect {
     fn collect_metrics_works() {
```

Comment:
`collect_metrics_contains_thread_pool_and_wallet_service_statistics`?

And if that name looks too long - probably it is a sign that this need to be split into two separate tests ;)

Reply:
Changed to ```collect_metrics_contains_thread_pool_and_wallet_service_statistics```